system/datatypes/peer_reviewed_article.php

```diff
@@ -1142,7 +1142,7 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
 		$html = <<<EOF
 			<div class="new-comment $type">
 			<div>
-				<strong><label for="$commentFieldName"> $comment </label></strong><textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
+				<strong><label for="$commentFieldName"> $comment </label></strong><textarea name="$commentFieldName" id="$commentFieldName"></textarea>
 			</div>
```

Comment:
The id that is produced here cannot be handled with css. 

What is produced:

<div>
--
```
 <strong><label for="discussion_comment/new_public"> reageer op het artikel </label></strong><textarea name="discussion_comment/new_public" id="discussion_comment/new_public"></textarea>
 </div>
```
In my opinion the id="discussion_comment new_public" would be better, since the slash produces an error in my css editor.

Reply:
Decided to delete the ID, since it was not needed.